smartsim/database/orchestrator.py

```diff
@@ -132,32 +133,27 @@ def create_cluster(self):  # cov-wlm
         self.check_cluster_status()
         logger.info(f"Database cluster created with {str(len(self.hosts))} shards")
 
-    def check_cluster_status(self):  # cov-wlm
+    def check_cluster_status(self):
         """Check that a cluster is up and running
 
         :raises SmartSimError: If cluster status cannot be verified
         """
-        # TODO use smartredis for this, then we don't have to create host dictionary
-        host_list = []
-        for host in self.hosts:
-            for port in self.ports:
-                host_dict = dict()
-                host_dict["host"] = host
-                host_dict["port"] = port
-                host_list.append(host_dict)
+        # pick a single host and port
+        host = self.hosts[0]
```

Comment:
What should we do if a user calls this method but the hosts and ports are not populated? like when the orchestrator is not launched. I tend to think we should have a try-except block here and throw a ``SmartSimError`` if hosts and ports are not populated.

this would also catch if the node aquisition strategy failed and ``check_cluster_status`` is still called.